runtime/onert/core/src/compiler/Compiler.cc

```diff
@@ -320,6 +480,226 @@ std::shared_ptr<exec::ExecutorMap> Compiler::compile(void)
+    _options.manual_scheduler_options.opcode_to_backend[ir::OpCode::If] = builtin_id;
+    _options.manual_scheduler_options.opcode_to_backend[ir::OpCode::While] = builtin_id;
+    _options.manual_scheduler_options.opcode_to_backend[ir::OpCode::Permute] = builtin_id;
+  }
+
+  // FIXME This is a workaround for bcq operations, should remove it
+  {
+    _options.manual_scheduler_options.opcode_to_backend[ir::OpCode::BCQFullyConnected] = "bcq";
+    _options.manual_scheduler_options.opcode_to_backend[ir::OpCode::BCQGather] = "bcq";
+  }
+
+  // It doesn't support tracing in case of partial graph
+  {
+    _options.tracing_ctx = nullptr;
+  }
+
+  {
+    VERBOSE(Compiler) << std::boolalpha << "==== Compiler Options ====" << std::endl;
+    VERBOSE(Compiler) << "backend_list             : "
+                      << nnfw::misc::join(_options.backend_list.begin(),
+                                          _options.backend_list.end(), "/")
+                      << std::endl;
+    VERBOSE(Compiler) << "trace_filepath           : " << _options.trace_filepath << std::endl;
+    VERBOSE(Compiler) << "graph_dump_level         : " << _options.graph_dump_level << std::endl;
+    VERBOSE(Compiler) << "executor                 : " << _options.executor << std::endl;
+    VERBOSE(Compiler) << "manual backend_for_all   : "
+                      << _options.manual_scheduler_options.backend_for_all << std::endl;
+    VERBOSE(Compiler) << "manual_scheduler_options : "
+                      << getOpBackends(_options.manual_scheduler_options.opcode_to_backend)
+                      << std::endl;
+    VERBOSE(Compiler) << "he_scheduler             : " << _options.he_scheduler << std::endl;
+    VERBOSE(Compiler) << "he_profiling_mode        : " << _options.he_profiling_mode << std::endl;
+    VERBOSE(Compiler) << "disable_compile          : " << _options.disable_compile << std::endl;
+    VERBOSE(Compiler) << "fp16_enable              : " << _options.fp16_enable << std::endl
+                      << std::noboolalpha;
+  }
+
+  _subgraphs->iterate([&](const ir::SubgraphIndex &, ir::Graph &subg) {
+    // Mandatory passes
+    auto part = subg.partialgraphs();
+    part->iterate([&](const ir::SubgraphIndex &, ir::Graph &partialgraph) {
+      pass::PassRunner{}
+        .append(std::make_unique<pass::ConstantOutputPass>(partialgraph))
+        .append(std::make_unique<pass::OddOutputPass>(partialgraph))
+        .run();
+
+      // Optimizations
+      pass::PassRunner{}
+        .append(std::make_unique<pass::UnusedOperandEliminationPass>(partialgraph))
+        .run();
+    });
+  });
+
+  /***************************************************
+   * Prepare compilation phase
+   ***************************************************/
+
+  // Compilable check
+  // TODO: Support hybrid execution -
+  //       execution between interpreter and compiled executor (including control flow)
+  if (_options.disable_compile)
```

Comment:
This option works with pipeline ?

Reply:
It works as the existing func without pipeline.